constraints-docker.txt

```diff
@@ -1,4 +1 @@
-datacube==1.8.3
-sqlalchemy==1.3.20
-click<8.0.0
-eodatasets3>=0.17.0
```

Comment:
There's appropriate minimum versions in the dependencies now, I believe. So I removed these.